### DISCUSSION TITLE
New Release?

### DISCUSSION THREAD

**Discussion by gerroon** [bpatrik/pigallery2#676] [General]
Hi

Is there a plan for a new release? It looks it has been like 6 months since the last docker release.

thanks

**Comment by bpatrik** [comment]
Not in the next few months I think, but not yet sure

-- Sorry for being brief, sent from my phone.

**Comment by bpatrik** [comment]
for those who also come by this:
I make about 1-2 releases per yer. (I plan to speed that up after the next 1-2 release. I wont mean more feature, but just faster checkpoints.)
If you would like to use the the latest features just go with the nightly build. I'm also using that so its not broken or anything. 
It is true tho that it might have more bugs, but should not be deal breaker. Its pretty hard to get something to the nightly release: 1) it should pass all the tests, 2) be able build 3) be able to build on docker.  +1 if there is a significant bug, I fix it within few days as It would also bother me
